tests/utils/test_batch_helpers.py

```diff
@@ -26,35 +26,24 @@ def __init__(self, **kwargs):
     counter_list.extend(num * [char])
 
 
-@pytest.fixture(scope="module",
-                params=[
-                    my_list,
-                    tuple(my_list),
-                    torch.tensor(my_list),
-                    np.asarray(my_list),
-                    deque(my_list),
-                    my_named_tuple(*my_list)
-                ])
+@pytest.fixture(scope="module", params=[
```

Comment:
Wow -- seems like a big refactor for the test cases. It's a bit hard to read through. Is it possible to minimize the diff here to the new test cases to handle the callables, or are some additional edge cases being added?

Reply:
Yeah, sorry, I did a lot of clean up and reorganization of the tests.